dev/com.ibm.ws.jsp.2.3_fat/fat/src/com/ibm/ws/jsp23/fat/tests/JSPTests.java

```diff
@@ -260,6 +261,7 @@ public void testEL30PropertyNotWritableException() throws Exception {
      */
     @Test
     @ExpectedFFDC("javax.el.PropertyNotWritableException")
+    @SkipForRepeat(SkipForRepeat.EE10_FEATURES)
```

Comment:
In the PR comments you say these are skipped due to changes in the Pages API , can you point me to the issue where we'll investigate these and work to reenable / update the tests for the Pages 3.1 feature?

Reply:
https://github.com/OpenLiberty/open-liberty/issues/21743